projects/vdk-plugins/vdk-postgres/src/vdk/plugin/postgres/ingest_to_postgres.py

```diff
@@ -57,28 +58,19 @@ def ingest_payload(
     @staticmethod
     def _populate_query_parameters_tuple(
         destination_table: str, cursor: PEP249Cursor, payload: List[dict]
-    ) -> (str, Tuple[str]):
+    ) -> (str, list):
         """
-        Returns insert into destination table tuple of query and parameters;
-        E.g. for a table dest_table with columns val1, val2 and payload size 3, this method will return:
-        'INSERT INTO dest_table (val1, val2) VALUES (%s, %s), (%s, %s), (%s, %s)', ['val1', 'val2']
-
-        :param destination_table: str
-            the name of the destination table
-        :param cursor: PEP249Cursor
-            the database cursor
-        :param payload: List[dict]
-            the payloads to be ingested
-        :return: Tuple[str, Tuple[str]]
-            tuple containing the query and parameters
```

Comment:
why delete the docstring?

Reply:
The param comments are redundant.

 But the description was good. I returned it (after adjusting it to the new interface).